tests/01http-server.pl

```diff
@@ -108,6 +110,8 @@
 # List of Awaiter structs
 my @pending_awaiters;
 
+my @default_responders;
```

Comment:
should this not be scoped to the HTTPServer instance?

Reply:
I guess??? The awaiters aren't but I don't know why